<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pseudonym</class>
 <widget class="QWidget" name="Pseudonym">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Псевдоним</string>
  </property>
  <widget class="QLabel" name="stones_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Задать количество камней</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="stones">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border-radius: 10px;
border: 3px solid #b7b7b7;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string>Задать</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="remainLcd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>541</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0, 0, 0, 0);
border-radius: 10px;
border: 3px solid #b7b7b7;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="takeInput_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Сколько камней взять?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="takeInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>129</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0, 0, 0, 0);
border-radius: 10px;
border: 3px solid #b7b7b7;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="takeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string>Взять</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>541</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border-radius: 5px;
border: 3px solid #b7b7b7;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0, 0, 0, 0);
border-radius: 10px;
border: 3px solid #b7b7b7;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
